src/app/pages/marketplace/style.scss

```diff
@@ -38,26 +42,22 @@
   .components-tab-panel__tabs {
     display: flex;
     /* <= WP 5.4 */
-    flex-direction: column;
+    flex-direction: row;
     /* <= WP 5.6 */
-    gap: 1rem;
+    gap: 1.5rem;
     min-width: 10rem;
     max-width: max-content;
     height: fit-content;
-    margin-top: .5rem;
-    margin-right: 2rem;
-    padding: .8rem .8rem .8rem 0;
+    margin: 0 .5rem 1rem .5rem;
```

Comment:
Let's remove this left margin and get the content to align with the header better? 
I think setting it to `0 .5rem 1rem 0` and setting the `.grid` `padding` to `0.5rem 0` would do it.

Reply:
Agreed, fixed.